scripts/exgdas_enkf_update.sh

```diff
@@ -287,7 +288,7 @@ cat > enkf.nml << EOFnml
    univaroz=.false.,adp_anglebc=.true.,angord=4,use_edges=.false.,emiss_bc=.true.,
    letkf_flag=${letkf_flag},nobsl_max=${nobsl_max},denkf=${denkf},getkf=${getkf}.,
    nhr_anal=${IAUFHRS_ENKF},nhr_state=${IAUFHRS_ENKF},
-   lobsdiag_forenkf=$lobsdiag_forenkf,
+   lobsdiag_forenkf=$lobsdiag_forenkf,taperanalperts=$taperanalperts,
```

Comment:
This will make the shellnorms test pass.
```suggestion
   lobsdiag_forenkf=${lobsdiag_forenkf},taperanalperts=${taperanalperts},
```

Reply:
Thanks for the suggestion @aerorahul.